avogadro/rendering/geometryvisitor.cpp

```diff
@@ -187,4 +189,55 @@ void GeometryVisitor::average()
   }
 }
 
+void GeometryVisitor::boundingBox(double& minX, double& minY, double& minZ,
```

Comment:
As I mentioned, it seems easier to put this into `core/molecule.*`. If there's some reason you can't access the Molecule class, this is okay.